projects/ngx-datatable/src/lib/components/body/body-row-wrapper.component.ts

```diff
@@ -27,55 +30,57 @@ import { DatatableRowDetailDirective } from '../row-detail/row-detail.directive'
   selector: 'datatable-row-wrapper',
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div
-      *ngIf="groupHeader && groupHeader.template"
-      [style.height.px]="groupHeaderRowHeight"
-      class="datatable-group-header"
-      [ngStyle]="getGroupHeaderStyle()"
-    >
-      <div class="datatable-group-cell">
-        <div *ngIf="groupHeader.checkboxable">
-          <label class="datatable-checkbox">
-            <input
-              #select
-              type="checkbox"
-              [checked]="selectedGroupRows.length === group.value.length"
-              (change)="onCheckboxChange(select.checked)"
-            />
-          </label>
+    @if (groupHeader && groupHeader.template) {
+      <div
+        [style.height.px]="groupHeaderRowHeight"
+        class="datatable-group-header"
+        [ngStyle]="getGroupHeaderStyle()"
+      >
+        <div class="datatable-group-cell">
+          @if (groupHeader.checkboxable) {
+            <div>
```

Comment:
I have not tested it, but do we need this empty div? This makes two divs

Reply:
we need it.